user/useradm.go

```diff
@@ -102,6 +103,22 @@ func NewUserAdm(jwtHandler jwt.Handler, db store.DataStore,
 	}
 }
 
+func (u *UserAdm) HealthCheck(ctx context.Context) error {
```

Comment:
just a comment: I cant shake the feeling that we should check something about users collection, maybe get users and see if we do not get an error?

Reply:
I don't think so, I strongly believe checking if the database is alive is enough.
Checking if the data is there is way too much for a health check, it's more a use-case for application monitoring (e.g. NewRelic, Zabbix, or similar). We already have application monitoring in NewRelic, no need to replicate it in the health check.

Reply:
what I meant is just call any of the GetUser calls (or similar) and expect either no error or no records error. But I can live without it. The point was: the health of the useradm depends on the healthy user collections. E.g.: see problems we had that people could not login. But not a problem if you want to keep only the db check.

Reply:
I changed the Ping to run the command on the "users" database instead of "admin", though I don't know if that's going to make much of a difference. I think testing each collection might become awkward on other services so I'd try to avoid it.

Reply:
even better